parseCSVFiles.py

```diff
@@ -59,6 +60,8 @@
 
 OVERFLOW_XLS_FILES = settings.OVERFLOW_XLS_FILES
 
+CSV_CLEAN_FILE_LIST = []
```

Comment:
this is global state, is there a way to make it smaller in scope, for example as a field of an object?

Reply:
The globals are not ideal. What I'm hoping is this particular library will not be around much longer, and may these can be ignored for now? Improvements can be made in the newer https://github.com/elifesciences/ejp-csv-parser/blob/develop/ejpcsvparser/csv_data.py  module of the other parsing library, where there are still some globals but some improvement in how the settings are specified, with work still to be done.